src/Dependency.php

```diff
@@ -52,4 +52,16 @@ public function hasDependency(string $dependency): bool
     {
         return in_array($dependency, $this->getDependencies(), true);
     }
+
+     /** @psalm-suppress PossiblyUnusedMethod */
```

Comment:
Any clue why this annotation is necessary?

Reply:
> Any clue why this annotation is necessary?

Yes, the psalm give error, because think it is not need,. I think some problem with psalm. I think we need to leave.